docs/hazmat/primitives/asymmetric/serialization.rst

```diff
@@ -521,7 +521,14 @@ file suffix.
     :type cert: :class:`~cryptography.x509.Certificate` or ``None``
 
     :param cas: An optional set of certificates to also include in the structure.
-    :type cas: list of :class:`~cryptography.x509.Certificate` or ``None``
+        If a :class:`~cryptography.hazmat.primitives.serialization.pkcs12.PKCS12Certificate`
+        is given, its friendly name will be serialized.
+    :type cas: list of
+        :class:`~cryptography.x509.Certificate`
+        ,
```

Comment:
I've read the current version as `list of (X or Y or None)` and not as `(list of (X or Y)) or None`. Maybe
```suggestion
        or
```
improves this?

Reply:
Perhaps one of the few places an oxford comma makes things less clear?